R/metadata.R

```diff
@@ -103,7 +104,7 @@ metadata <- function(surveyID,
   # QUERY API ----
 
   # Function-specific API stuff
-  survey_url <- create_survey_url(Sys.getenv("QUALTRICS_BASE_URL"), surveyID)
+  survey_url <- create_survey_url(Sys.getenv("QUALTRICS_BASE_URL"), surveyID, survey_definition)
```

Comment:
Use the new `endpoint` variable to create the URL; the `create_survey_url()` function can be changed to paste together the endpoint with the other info.